Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -48,6 +48,9 @@ def bump_formula_pr_args
              description: "Print the pull request URL instead of opening in a browser."
       switch "--no-fork",
              description: "Don't try to fork the repository."
+      switch "--automerge",
+             env:         :bump_automerge,
+             description: "Open pull requests with automerge and branch cleanup enabled."
```

Comment:
Not sure this works as-is yet, since `bump-formula-pr` PRs will still need a bottle commit pushed, so I'm not sure we want to enable automerge on them. In theory doing so should do nothing given the bottle checks in the merge queue though.

Reply:
@carlocab Oops, I've misunderstood the workflow. Does this make this PR basically pointless for homebrew/core (and/or homebrew/core)?

Reply:
At the moment, it is pretty pointless. I can see this being useful if/when PRs are blocked from being queued to merge if they are still missing a bottle commit (and require one).

Reply:
@Homebrew/cask would this be useful for homebrew/cask?

Reply:
Cask has automation for this I believe

Reply:
Yeah, homebrew/cask* already auto-enables automerge for all maintainer-authored PRs labelled `bump-cask-pr`: https://github.com/Homebrew/homebrew-cask/blob/master/.github/workflows/automerge.yml.

Example: https://github.com/Homebrew/homebrew-cask/pull/152038

Reply:
Cool, problem solved then, thanks all.